flask_backend_design.py

```diff
@@ -0,0 +1,272 @@
+""" importing required libraries for this package """
+from flask import Flask, request, jsonify
+from flask import redirect, url_for
+import pyodbc
+import datetime
+import os
+date = datetime.datetime.now()
+CurrentUser = os.getlogin()
+
+app = Flask(__name__)
+
+""" Establishing connection to database"""
+
+
+def init_db():
+    con = pyodbc.connect("Driver={ODBC Driver 13 for SQL Server}; \
```

Comment:
Connection string can be placed as constant.
Params like id, password, server etc can be abstracted to another module, say settings.py, which will enable customisation.